akka-cluster-sharding-typed/src/test/scala/akka/cluster/sharding/typed/scaladsl/ClusterShardingSpec.scala

```diff
@@ -304,18 +304,14 @@ class ClusterShardingSpec
 
       peterRef ! StopPlz()
 
-      // FIXME #26514: doesn't compile with Scala 2.13.0-M5
-      /*
       // make sure request with multiple parameters compile
       Behaviors.setup[TheReply] { ctx =>
-        ctx.ask(aliceRef)(WhoAreYou2(17, _)) {
+        ctx.ask(peterRef, WhoAreYou2(17, _)) {
```

Comment:
sorry, for Scala 3 it has to be:
```
ctx.ask[WhoAreYou2, String](peterRef, WhoAreYou2(17, _)) {
```

otherwise it doesn't understand that the reply is String

Reply:
@patriknw  Thank you for your advice :)